daprdocs/content/en/operations/hosting/kubernetes/kubernetes-deploy-without-cli.md

```diff
@@ -0,0 +1,299 @@
+      containers:
+      - command:
+        - ./daprd
+        - --mode
+        - orchestrator
+        - --app-id
+        - application
+        - --app-port
+        - "3000"
+        - --placement-host-address
+        - dapr.dapr.svc:50005
+        - --dapr-grpc-port
+        - "50001"
+        - --dapr-internal-grpc-port
+        - "50002"
+        - --dapr-http-port
+        - "3500"
+        - --app-protocol
+        - http
+        - --log-level
+        - debug
+        - --components-path
+        - /components
+        image: daprio/daprd:edge
+        name: application-edge
+        ports:
+        - containerPort: 3500
+          name: dapr-http
+          protocol: TCP
+        - containerPort: 50001
+          name: dapr-grpc
+          protocol: TCP
+        - containerPort: 50002
+          name: dapr-internal
+          protocol: TCP
+        resources:
+          limits:
+            cpu: 200m
+            memory: 200Mi
+          requests:
+            cpu: 200m
+            memory: 200Mi
+        volumeMounts:
+        - mountPath: /components/statestore.yaml
+          name: application-components-statestore
+          subPath: statestore.yaml
+      - env:
+        - name: ACTOR_IDLE_TIMEOUT
+          value: "100"
+        - name: ACTOR_SCAN_INTERVAL
+          value: "10"
+        - name: ACTOR_DRAIN_ONGOING_CALL_TIMEOUT
+          value: "10"
+        - name: ACTOR_DRAIN_BALANCED
+          value: "true"
+        - name: DAPR_ZIF_UDL_PROVIDER_NAME
+          value: zif-cm-provider-udl-service
+        - name: DAPR_ZIF_UDL_PROVIDER_CLIENTS
+          value: "5"
+        - name: DAPR_HTTP_PORT
+          value: "3500"
+        - name: DAPR_GRPC_PORT
+          value: "50001"
+        image: application:latest
+        name: application
+        ports:
+        - containerPort: 3000
+        resources:
+          limits:
+            cpu: 200m
+            memory: 200Mi
+          requests:
+            cpu: 200m
+            memory: 200Mi
+      enableServiceLinks: false
+      imagePullSecrets:
+      - name: zyfra
+      volumes:
+      - configMap:
+          defaultMode: 420
+          name: application-components-statestore
+        name: application-components-statestore
+
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    description: Exposes and load balances the application pods
+  labels:
+    app: application
+    name: application
+  name: application-dapr
+spec:
+  ports:
+  - name: dapr-http
+    port: 80
+    protocol: TCP
+    targetPort: 3500
+  - name: dapr-grpc
+    port: 50001
+    protocol: TCP
+    targetPort: 50001
+  - name: dapr-internal
+    port: 50002
+    protocol: TCP
+    targetPort: 50002
+  selector:
+    name: application
+```
+The next example shows how to apply it to Kubernetes.
+
+```bash
+kubectl apply -f application.yml
+```
+As you can see, to run application we use mode named **dns**. In this mode to locate the Dapr application we use services with suffix `-dapr`.
```

Comment:
Suggest reword this to
"To run the application use the **dns** mode . In this mode to locate the Dapr application, use services with suffix `-dapr`.